2.0/s2i/bin/run

```diff
@@ -10,6 +10,16 @@ set -e
 export RACK_ENV=${RACK_ENV:-"production"}
 
 if is_puma_installed; then
+  export_str=$(cgroup-limits.py)
+  [[ -n "$export_str" ]] && export $export_str
+
+  if [[ -z "${MEMORY_LIMIT_IN_BYTES:-}" ]]; then
+    echo "Warning: Can't detect memory limit"
+  fi
+  if [[ -z "${NUMBER_OF_CORES:-}" ]]; then
+    echo "Warning: Can't detect number of cores"
```

Comment:
Maybe just update the warning with a little more info, like "Can't detect number of cores, defaulting setting FOO to BAR"

that way we still get the warning (for when we are debugging user environments) but users who see it will have a warm fuzzy about the implication of the warning.

You can even print "Warning: can't detect number of cores.  Setting FOO to BAR, set env variable XXX to override"

(and we should only print the warning if they haven't already explicitly set the values that would be set based on the cgroup value.  That is, if we weren't going to do anything with the cgroup value anyway, then it's not worth warning them about it)